### PR TITLE
Asciidoctor migration

### PR DESCRIPTION
**Pull request by dedemorton**
First pass - removes complex conditionals, but there are still many errors when I build this (and it looks like a lot of the errors are caused by asciidoc files that we generate).

### CODE REVIEW

**Comment by nik9000** [comment]
I haven't checked the logic but I love this way of removing the complex
conditionals!

**Comment by dedemorton** [comment]
Incidentally, if you find out that the problems are caused by the formatting that we use in the "Exported fields" topic...we would like to change the layout anyhow (could be a win-win). Here's what I've proposed: https://github.com/elastic/beats/issues/9519  (POC is here: https://github.com/dedemorton/beats-docs-review/pull/2/files). We should figure out what they've done with ECS and build on that. Eventually we want to pull a lot of the descriptions from the ECS docs, but not sure we want to get all those things figured out before we migrate to asciidoctor. Just an FYI....

File: journalbeat/docs/index.asciidoc [hunk 1]
```diff
@@ -13,6 +13,7 @@ include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
 :github_repo_name: beats
 :discuss_forum: beats/{beatname_lc}
 :beat_default_index_prefix: {beatname_lc}
+:has_script_processor:
```
Comment (nik9000) [inline journalbeat/docs/index.asciidoc]: I think this is missing ` :has_decode_csv_fields_processor:`.

File: libbeat/docs/command-reference.asciidoc [hunk 2]
```diff
@@ -82,31 +82,34 @@ endif::[]
...
 |<<export-command,`export`>> |{export-command-short-desc}.
 |<<help-command,`help`>> |{help-command-short-desc}.
 |<<keystore-command,`keystore`>> |{keystore-command-short-desc}.
-ifeval::[("{beatname_lc}"=="functionbeat")]
+ifeval::["{beatname_lc}"=="functionbeat"]
 |<<package-command,`package`>> |{package-command-short-desc}.
 |<<remove-command,`remove`>> |{remove-command-short-desc}.
 endif::[]
-ifeval::[("{beatname_lc}"=="filebeat") or ("{beatname_lc}"=="metricbeat")]
+ifeval::["{beatname_lc}"=="filebeat"]
```
Comment (nik9000) [inline libbeat/docs/command-reference.asciidoc]: Should this be `ifdef::has_modules_command[]`?

File: journalbeat/docs/index.asciidoc (hunk 1 above)
Comment (dedemorton) [reply journalbeat/docs/index.asciidoc]: you're right! I forgot to push the commit where I've updated this.

File: libbeat/docs/command-reference.asciidoc (hunk 2 above)
Comment (dedemorton) [reply libbeat/docs/command-reference.asciidoc]: yup! You're right. I created the attribute to deal with NOT conditions, but I can use it here, too. Wheeeee.

**Pull request merged**